docs/content/Installation/postgres-operator-installer/_index.md

```diff
@@ -50,4 +50,40 @@ the Crunchy PostgreSQL Operator that you are installing.
 The image pull policy needs to be defined for your job.
 In most cases this should be updated to `IfNotPresent`.
 
+### Running the pgo-client
```

Comment:
There should also be some overall explanation of what the pgo Client is (you can borrow a one-liner from elsewhere) and they there are different ways to install them, and why you may choose one over the other.